release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -3152,7 +3152,7 @@ methods and strategies] for instructions.
 
 Issued: 2020-03-18
 
-An update for jenkins-slave-base-rhel7-container is now available for {product-title} 3.11.
+An update for `jenkins-slave-base-rhel7-container` is now available for {product-title} 3.11.
```

Comment:
Side note: I'm curious if this container name will be updated with the language updates in the works.